### DISCUSSION TITLE
How Many Server required if we are onboarding 10000 Users at a time in Exo

### DISCUSSION THREAD

**Discussion by paljitender** [exoplatform/exo-community#72] [Q&A]
Hi Team,

If we 10000 users going to use exo then what will be the system configuration is required. Please suggest.

Let's assume we are going to use AWS cloud and going to use Ec2 instances then how many instances would be needed.

Thanks,
Jitender Pal

**Comment by rdenarie** [comment]
Hello,
Can you register on https://community.exoplatform.com/
and ask your question in eXo Community Space ?
You will have more chance to get an answer

**Comment by paljitender** [comment]
This site is not working, Could you please suggest here as it is urgent.

Please find below the screen shot while hitting the URL you suggested.

![image](https://github.com/exoplatform/exo-community/assets/113160963/3f894387-128e-4668-aa9b-639ea95a0daa)

**Comment by rdenarie** [comment]
The site is now working. Can you retest ? 

In fact, your question cannot be answered as is. It depends and a lot of factor, and I will not be able to anwser directly here. 
On Tribe, there are other people which will be able to orient you.
